.github/workflows/build_test.yaml

```diff
@@ -35,15 +35,14 @@ jobs:
       - name: Build
         run: |
           cd substrate-node
-          rustup target add wasm32-unknown-unknown --toolchain nightly-2022-05-11
-          cargo +nightly-2022-05-11 build --release
+          cargo build --release
 
       - name: Unit tests
         run: |
           cd substrate-node
-          cargo +nightly-2022-05-11 test --no-fail-fast
+          cargo test --no-fail-fast
```

Comment:
Not sure how to do that but we could also include the benchmark auto-generated unit tests somewhere by doing something like this:
~~~ 
cargo test --features runtime-benchmarks
~~~